drivers/gnss/gnss_luatos_air530z.c

```diff
@@ -336,14 +336,14 @@ static int luatos_air530z_get_supported_systems(const struct device *dev, gnss_s
 	return 0;
 }
 
-static struct gnss_driver_api gnss_api = {
+static const struct gnss_driver_api gnss_api = {
 	.set_fix_rate = luatos_air530z_set_fix_rate,
 	.set_enabled_systems = luatos_air530z_set_enabled_systems,
 	.get_supported_systems = luatos_air530z_get_supported_systems,
 };
 
 #define LUATOS_AIR530Z(inst)								\
```

Comment:
needs proper reformatting (one more tab needed) so that all `\` are properly right-aligned

Reply:
Got it.